### DISCUSSION TITLE
Best practices copying productive instance to a test instance

### DISCUSSION THREAD

**Discussion by LFrank2021** [opendcim/openDCIM#1384] [Q&A] [answered]
Hi, is there already a best practice on how to duplicate a productive instance to a testing instance?
I assume:
prepare httpd server maybe with apache auth first
install the binaries or copy the entire directory (modifying db.inc.php)
restore the database from a backup
There seem to be some attributes in the fac_Config that could use some modifications but are those necessary?

Kind regards
Frank

**Comment by samilliken** [comment, marked as answer]
update fac_Config set Value='https://your.new.url' where Parameter='InstallURL';

Also, if you are using SAML, change (at minimum - assuming you use the same IdP profile):
update fac_Config set Value='https://your.new.url' where Parameter='SAMLBaseURL';
